bootstrap-cmake.sh

```diff
@@ -33,3 +61,4 @@ cmake3 \
     ${EXTRA_CMAKE_ARGS} \
     -DVESPA_LLVM_VERSION=$VESPA_LLVM_VERSION \
     "${SOURCE_DIR}"
+fi
```

Comment:
For code de-duping consider creating a `$VESPA_INSTALL_PREFIX` variable dependent on `$UNPRIVILEGED` and substituting that for `/opt/vespa` vs `$HOME/vespa` and also possibly appending VESPA_USER/VESPA_UNPRIVILEGED stuff to `EXTRA_CMAKE_ARGS`